src/root.jl

```diff
@@ -486,6 +481,7 @@ function readbasketseek(f::ROOTFile, branch::Union{TBranch, TBranchElement}, see
     basketkey = unpack(rawbuffer, TBasketKey)
     compressedbytes = compressed_datastream(rawbuffer, basketkey)
 
+    @debug "Seek position: $seek_pos"
```

Comment:
I have recently come to understand that `@debug` is not technically zero-cost, maybe we should do a round of cleaning of all `@debug` in inner code

Reply:
Oh wow, didn't know that. Yes we should do that indeed 😉 Where did you find that?

Reply:
https://docs.julialang.org/en/v1/stdlib/Logging/

>very cheap

it's a run-time check to see if we should run the debug block